src/scripts/calculation_download_gdrive_info.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import json
+import math
+import os
+import typing
+from urllib import request
+
+import pandas as pd
+
+
+def replace_nan(value, replace):
```

Comment:
Are there any guidelines about how functions should be sorted in a file? I was sort of expecting these to be in alpha order but they aren't so I'm not sure what the best practice is. Also, seeing a class defined in the middle of the file surprised me as well but I'm really not all that familiar with how the contents of a module should be organized.

Reply:
As far as I know, there are no guidelines about that. Personally, I don't stick to anything strictly, though I would be happy to start doing so. I don't know what is best practice specifically however. @tommy-stone is there any particular convention for this?